src/Kunstmaan/MediaBundle/EventListener/DoctrineMediaListener.php

```diff
@@ -80,7 +80,7 @@ public function preUpdate(PreUpdateEventArgs $eventArgs)
                 if ($deleted || $reverted) {
                     $oldFileUrl = $entity->getUrl();
                     $newFileName = ($reverted ? $entity->getOriginalFilename() : uniqid());
-                    $newFileUrl = dirname($oldFileUrl) . '/' . $newFileName . '.' . pathinfo($oldFileUrl, PATHINFO_EXTENSION);
+                    $newFileUrl = dirname($oldFileUrl).'/'.$newFileName.'.'.pathinfo($oldFileUrl, PATHINFO_EXTENSION);
```

Comment:
Because that's the coding standard ;)

> Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;